ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -168,6 +168,43 @@ def test_mean(
     )
 
 
+# median
```

Comment:
You can also just remove this from the PR

Reply:
It has been removed